code/modules/surgery/surgery_step.dm

```diff
@@ -175,3 +175,8 @@
 	if(!target_detailed)
 		var/you_feel = pick("a brief pain", "your body tense up", "an unnerving sensation")
 		target.show_message(vague_message, MSG_VISUAL, span_notice("You feel [you_feel] as you are operated on."))
+
+/datum/surgery_step/proc/display_pain(mob/living/target, pain_message, mechanical_surgery = FALSE)
```

Comment:
make it early return if youre unconscious?

Reply:
I did have an unconsciousness check, I must've lost it in the port